src/main.js

```diff
@@ -214,27 +213,28 @@ function savePosterToGrid() {
   savePoster(currentPoster);
 }
 
-function goToMain() {
-  hideForm();
-  showMainPoster();
-  hideSavedPosters();
-}
-//refactor remove/add to toggle?
-function showForm() {
-  form.classList.remove("hidden");
+function deletePoster() {
+  //poster will be deleted from grid
+  //target ---> savedPostersGrid
+  //target class ---> "mini-poster"
+  if (event.target.classList.contains("mini-poster")) {
+      console.log("pls work")
+    for (var i = 0; i < savedPosters.length; i++) {
+      if (event.target.id === savedPosters[i].id){
```

Comment:
I believe if you were to console log `event.target` in this `for` block, you will see that you will not be able to directly access `id` from it!

Reply:
Because right now, `id` is on a `div` that within the target `section`, not on the `section` itself.

Reply:
Oh, I see! So I need to put the id on the section, rather than the div.  Thank you!